src/desktop/apps/conversations/client.tsx

```diff
@@ -0,0 +1,32 @@
+import { buildClientApp } from "reaction/Artsy/Router/client"
+import { data as sd } from "sharify"
+import { conversationRoutes } from "reaction/Apps/Conversation/routes"
+import React from "react"
+import ReactDOM from "react-dom"
+import { setupArtistSignUpModal } from "desktop/apps/artist/components/cta"
+
+const mediator = require("desktop/lib/mediator.coffee")
+
+buildClientApp({
+  routes: conversationRoutes,
+  context: {
+    user: sd.CURRENT_USER,
+    mediator,
+  } as any,
+})
+  .then(({ ClientApp }) => {
+    ReactDOM.hydrate(
```

Comment:
This will want to be wrappd in a [loadableReady function](https://github.com/artsy/force/blob/master/src/desktop/apps/experimental-app-shell/client.tsx#L36) so that split bundles can load before the page renders.